File: 2095-minimum-number-of-swaps-to-make-the-string-balanced/2095-minimum-number-of-swaps-to-make-the-string-balanced.cpp
class Solution {
public:
    int minSwaps(string s) {
        stack<int> st;

        for(auto c : s){
            if(c == '[') st.push(c);
            else if(!st.empty()) st.pop();
        }
        return (st.size()+1)/2;
    }
};

File: 2095-minimum-number-of-swaps-to-make-the-string-balanced/2095-minimum-number-of-swaps-to-make-the-string-balanced.py
class Solution(object):
    def minSwaps(self, s):
        """
        :type s: str
        :rtype: int
        """
        stack = []
        for i in s :
            if i == '[':
                stack.append(i) 
            elif stack:
                stack.pop()

        return (len(stack) + 1) // 2 
        """to get interger value without fraction part"""
        